sdk/python/touca/cli/server.py

```diff
@@ -3,38 +3,64 @@
 import logging
 from argparse import ArgumentParser
 from pathlib import Path
-from typing import List, Union
 
 from touca.cli.common import CliCommand
 
 logger = logging.getLogger("touca.cli.server")
 
 
-def check_prerequisites():
-    from shutil import which
-
-    requirements = {
-        "docker": (
-            "We use Docker to install Touca server."
-            " We could not find it on your system."
-            " Please install docker and try this script again."
-            " See https://docs.docker.com/get-docker/ for instructions."
-            " Have a great day!"
-        ),
-        "docker-compose": (
-            "We use docker-compose to install Touca server."
-            " We could not find it on your system."
-            " Please install docker-compose and try this script again."
-            " See https://docs.docker.com/compose/install/ for instructions."
-            " Have a great day!"
-        ),
-    }
-    for cmd, error in requirements.items():
-        if not which(cmd):
-            raise RuntimeError(error)
-
-
-def find_compose_file(install_dir) -> Union[Path, None]:
+class Compose:
```

Comment:
The reason for creating this class was to enable using `docker_compose_exec` inside `run()` so we don't have to decide between `docker compose` and `docker-compose` every time we want to run a docker compose command. The alternative was to use a module-level variable.